docs/lcb/notes/feature-objc_blocks.md

```diff
@@ -0,0 +1,79 @@
+# LiveCode Builder Language
+
+## Objective-C block support
+
+The handler `CreateObjcBlockPointerFromHandler` has been added the `objc` module
+which facilitates the calling of Objective-C methods that require blocks.
+Objective-C blocks are objects defining fragments of code which can be called at
+a later date, typically used by methods that require progress callbacks or
+completion handlers.
+
+`CreateObjcBlockPointerFromHandler` wraps a LCB handler in an `ObjcBlockPointer`
+and returns true on success and false otherwise. In order to call the
+`CreateObjcBlockPointerFromHandler`, pass the handler to be wrapped as the first
+argument and the variable into which the `ObjcBlockPointer` should be placed as
+the second.
+
+	private variable sRequestPermissionsCompletionHandler as optional ObjcBlockPointer
```

Comment:
Does this need to be in a triple-backquote block (I presume our markdown parser handles that...)?

Reply:
I've followed a similar structure to what we have in https://github.com/livecode/livecode/blob/develop/docs/lcb/notes/feature-objc_delegates.md which uses indents for code blocks. I guess that is maybe what the guide generator is expecting.

The file is rendered correctly by GitHub https://github.com/livecode/livecode/blob/develop/docs/lcb/notes/feature-objc_delegates.md